csrankings-p.csv

```diff
@@ -1097,8 +1097,7 @@ Purushotham Bangalore,University of Alabama - Birmingham,https://www.cis.uab.edu
 Purushotham V. Bangalore,University of Alabama - Birmingham,https://www.cis.uab.edu/puri,NOSCHOLARPAGE
 Purushottam Kar,IIT Kanpur,http://www.cse.iitk.ac.in/users/purushot,MNXz0AoAAAAJ
 Purushottam Kulkarni,IIT Bombay,https://www.cse.iitb.ac.in/~puru,Gs3HO5kAAAAJ
-Pushpak Bhattacharya,IIT Patna,https://www.cse.iitb.ac.in/~pb,vvg-pAkAAAAJ
-Pushpak Bhattacharyya,IIT Patna,https://www.cse.iitb.ac.in/~pb,vvg-pAkAAAAJ
+Pushpak Bhattacharyya,IIT Bombay,https://www.cse.iitb.ac.in/~pb,vvg-pAkAAAAJ

```

Comment:
Both entries must be here (there must be an entry for every DBLP alias).

Reply:
ah, I see. udpated  it.